lib/built-in-rules-index.js

```diff
@@ -0,0 +1,275 @@
+/**
+ * @fileoverview Collects the built-in rules into a map structure so that they can be imported all at once and without
+ * using the file-system directly.
+ * @author Peter (Somogyvari) Metz
+ */
+
+"use strict";
+
+module.exports = {
+    "no-constant-condition": require("./rules/no-constant-condition"),
```

Comment:
Can you sort these properties alphabetically? That should avoid most merge conflicts when people are adding rules to the list.

For convenience, [here](https://gist.github.com/not-an-aardvark/c95d7a38bec0171ac8553453142adbb6) is an updated version of the file with the keys sorted.

Reply:
Big thanks for the ordered version of the file. Updated it.